backend/codesirius/authentication/migrations/0011_alter_user_last_name.py

```
@@ -0,0 +1,18 @@
+# Generated by Django 5.1.5 on 2025-02-10 11:29
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("authentication", "0010_alter_user_last_name"),
+    ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)